hexa/pipelines/management/commands/pipelines_runner.py

```diff
@@ -33,6 +38,12 @@ def run_pipeline_kube(run: PipelineRun, env_var: dict):
         + "-"
         + exec_time_str
     )
+    pipeline_timeout = (
+        run.pipeline.timeout
+        if run.pipeline.timeout
+        else os.environ.get("PIPELINE_RUN_DEFAULT_TIMEOUT")
```

Comment:
Please use Django settings instead of accessing the env variable directly (`from django.cong import settings`)